GenericDiffractometer.py

```diff
@@ -117,14 +117,27 @@ class GenericDiffractometer(HardwareObject):
                    "kappa",
                    "kappa_phi",
                    "beam_x",
-                   "beam_y"]
+                   "beam_y",
+                   "back_light",
```

Comment:
Hi @JieNanMAXIV 

Sorry for disturbing, but MOTORS_NAME is a list to be used for centering positions (to create, compare, etc.). I think back_light and front_light should not be here.
What do you think?